ts/components/wallet/TransactionsList.tsx

```diff
@@ -121,8 +121,9 @@ export default class TransactionsList extends React.Component<Props, State> {
             amount,
             datetime: dateToAccessibilityReadbleFormat(
               item.created,
-              I18n.t("global.accessibility.dateTime_format")
+              I18n.t("global.accessibility.date_format")
```

Comment:
@Undermaken We can remove the explicit format here since it is already defined inside the `dateToAccessibilityReadbleFormat` definition